etl/src/main/java/org/pih/hivmigration/etl/sql/SqlMigrator.java

```diff
@@ -317,4 +321,68 @@ private String abbreviate(String str) {
         }
     }
 
+    // Methods that allow for creating tests that should pass
+
+    protected void assertAllRows(String description, String mysqlQuery) throws Exception {
+        log.info("Testing: " + description);
+        Number allRows = (Number) selectMysql("select count(*) from drug_order", new ScalarHandler<Number>());
```

Comment:
Looks like this function is specific to the `drug_order` table. Can you generalize it, maybe with a `tableName` parameter?